Makefile

```diff
@@ -41,7 +41,7 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="github.com/vmware-tanzu/nsx-operator/pkg/apis/crd.nsx.vmware.com/v1alpha1" output:crd:artifacts:config=build/yaml/crd/vpc/
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="github.com/vmware-tanzu/nsx-operator/pkg/apis/vpc/v1alpha1" output:crd:artifacts:config=build/yaml/crd/vpc/
```

Comment:
Are you seeing https://github.com/zhengxiexie/nsx-operator-1/tree/zhengxie/rename_stale/pkg/apis ?

Reply:
Since t1 is only for compatibility instead of going on developing, so we only consider VPC hence.

Reply:
I mean [#671](https://github.com/vmware-tanzu/nsx-operator/pull/671)
It looks like that you delete  Remove ippool CRD and apis/vpc/v1alpha2 in this patch instead of the patch https://github.com/vmware-tanzu/nsx-operator/pull/671

Reply:
Perhaps in this PR https://github.com/vmware-tanzu/nsx-operator/pull/675/files#diff-fb2882c620ac0e86e1379539b9bd69b91bdf3e702241b3e620bcfadf6c2787c2 which is a pre-requirement, add ippool by mistake.